src/vos/vos_space.c

```diff
@@ -50,31 +44,37 @@ vos_space_sys_init(struct vos_pool *pool)
 	daos_size_t	scm_tot = pool->vp_pool_df->pd_scm_sz;
 	daos_size_t	nvme_tot = pool->vp_pool_df->pd_nvme_sz;
 
-	POOL_SCM_SYS(pool) =
-		get_frag_overhead(scm_tot, DAOS_MEDIA_SCM, pool->vp_small);
-	POOL_NVME_SYS(pool) =
-		get_frag_overhead(nvme_tot, DAOS_MEDIA_NVME, pool->vp_small);
+	POOL_SCM_SYS(pool)  = get_frag_overhead(scm_tot, DAOS_MEDIA_SCM);
+	POOL_NVME_SYS(pool) = get_frag_overhead(nvme_tot, DAOS_MEDIA_NVME);
 
-	gc_reserve_space(&pool->vp_space_sys[0]);
-	agg_reserve_space(&pool->vp_space_sys[0]);
+	gc_reserve_space(&pool->vp_space_sys[DAOS_MEDIA_SCM]);
+	agg_reserve_space(&pool->vp_space_sys[DAOS_MEDIA_SCM]);
```

Comment:
It's better to have a D_INFO message to print the total & reserved space for both SCM and NVMe.

Reply:
- [ ] Replace DEBUG message with INFO

Reply:
Fixed with commit https://github.com/daos-stack/daos/pull/13013/commits/270805402e811b5f153193363cf4304b7dcc26f0#diff-d43f9e7f3650645f86cda7b2c96d5e007c6b0567bdce7807f25da2274fa5b42dL22